### DISCUSSION TITLE
Xbox Series X|S can be used?

### DISCUSSION THREAD

**Discussion by GeorgeCalderoESP** [cathery/sys-con#220] [Q&A]
Syscon can support XBOX Series S|X pad?

Thanks

**Comment by TheZnuf** [comment]
Just try it, It Works directly connected by USB. The button are "inversed" A=A etc...

Can't connect with the adapter, maybe I need a powered hub.

**Comment by GeorgeCalderoESP** [comment]
I can confirm it doesnt work.

I tried a PS4 controller and it worked but XBOX Series X|S doesnt work.

Xbox 360 works.

**Comment by TheZnuf** [comment]
Mine work wired with last firmware

**Comment by GeorgeCalderoESP** [comment]
> Mine work wired with last firmware

is it the new xbox Series X|S controller? 

I had a similar problem with Windows 10 and this new controller and the solution was to update Windows.

If I plug in the controller, the light turns on and the Switch recognizes the controller because the icon appears at the bottom left but it doesn't work

**Comment by TheZnuf** [comment]
Weird. It's didn't work anymore. Maybe the new pad firmware. I can see the icon but no touch reponse 😕

**Comment by cathery** [comment]
I have no idea how Xbox Series X/S controllers work, so I don't know if they're supported. I'll have to get one to see for myself. From the replies it seems to have the same issue as Xbox one controllers (#100) so maybe it's similar to an Elite controller.
